README.md

```diff
@@ -22,17 +22,17 @@ A common format for describing and capturing:
 1. workload configuration
 1. performance test results
 
-This specification is currently being advanced through the [Meshery](https://layer5.io/meshery) project. See a [screenshot](docs/assets/spec/readme/meshery_benchmark_screen.png) of its implementation.
+This specification is currently being advanced through the [Meshery](https://layer5.io/meshery) project. See a [screenshot](docs/assets/perf/readme/meshery_benchmark_screen.png) of its implementation.
 
 ## Example
 
-See the [sample results](docs/assets/spec/readme/example/service%20mesh%20performance%20specification%20result.yaml) file.
+See the [sample results](docs/assets/perf/readme/example/service%20mesh%20performance%20result.yaml) file.
```

Comment:
```suggestion
See the [sample results](docs/assets/perf/readme/service mesh performance result.yaml) file.
```
The link is broken here.

Reply:
@chandrashritii, good spot! committed your suggestion 🎈 
Thanks!